--- termpkg-3.2/libtn/SocketIO.c.orig	Sat Jan  8 19:20:55 2000
+++ termpkg-3.2/libtn/SocketIO.c	Wed Sep  5 14:22:46 2001
@@ -25,6 +25,7 @@
 #endif
 #include <sys/socket.h>
 #include <netinet/in.h>
+#include <arpa/inet.h>
 #include <fcntl.h>
 #include <string.h>
 #include <stdlib.h>
@@ -47,13 +48,6 @@
 
 int tn_errno = 0;
 
-#if !defined(AIX)
-#if !defined(FreeBSD) && !defined(SunOS)
-u_short htons();
-#endif
-u_short inet_addr();
-#endif
-
 static void timerExpired(int sig)
 {
    sig = 0;
diff -uNr termpkg-3.2.orig/termnetd/tndCfgParser.c termpkg-3.2/termnetd/tndCfgParser.c
--- termpkg-3.2.orig/termnetd/tndCfgParser.c	Sat Feb  3 16:04:14 2001
+++ termpkg-3.2/termnetd/tndCfgParser.c	Sun Aug 25 16:42:41 2002
@@ -82,7 +82,7 @@
 int xlate_cflag[] =
 {
    CS5, CS6, CS7, CS8, CSTOPB, PARENB, PARODD, HUPCL, CLOCAL,
-#if !defined(FreeBSD) && !defined(SCO) && !defined(AIX) && !defined(OSF)
+#if !defined(FreeBSD) && !defined(SCO) && !defined(AIX) && !defined(OSF) && !(defined(LINUX) && defined(__powerpc__))
    CIBAUD, CRTSCTS,
 #else
    0, 0,
